dll/win32/msutb/lang/pt-PT.rc

```diff
@@ -0,0 +1,79 @@
+/*
+ * PROJECT:     ReactOS msutb.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Portuguese (Portugal) resource file
+ * TRANSLATOR:  Copyright 2023-2024 Jose Carlos Jesus <zecarlos1957@hotmail.com>
+ */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE
+
+STRINGTABLE
+BEGIN
+    IDS_OK               "&OK"
+    IDS_CANCEL           "&Cancelar"
+    IDS_ABORT            "&Abortar"
+    IDS_RETRY            "&Retroceder"
+    IDS_IGNORE           "&Ignorar"
+    IDS_YES              "&Sim"
+    IDS_NO               "&Não"
+
+    IDS_SETTINGS         "&Definições..."
+    IDS_RESTORELANGBAR   "Restaurar a barra de idiomas"
+    IDS_LANGUAGE         "Linguagem"
+    IDS_LANGUAGEBUTTON   "Botão Linguagem"
+    IDS_TRANSPARENCY     "&Transparência"
+    IDS_TEXTLABELS       "T&exto da etiqueta"
+    IDS_MINIMIZE         "&Minimizar"
+    IDS_CLOSELANGBAR     "&Fechar a barra de idiomas"
+    IDS_EXTRAICONS       "&Ícones adicionais na barra de tarefas"
+    IDS_LANGUAGEBAR      "A barra de idiomas"
+    IDS_RESTORELANGBAR2  "&Restaurar a barra de idiomas"
+    IDS_MENUWND          "Menu Window"
+    IDS_VERTICAL         "&Vertical"
+    IDS_ADJUSTLANGBAND   "&Ajuste a posição da banda de idioma"
+    IDS_LEFTCLICK        "Clique esquerdo"
+END
+
+IDD_CLOSELANGBARNOBAND DIALOGEX 0, 0, 200, 105
+CAPTION "Barra de idiomas"
+STYLE WS_POPUPWINDOW | WS_CAPTION | DS_MODALFRAME
+FONT 8, "MS Shell Dlg"
+{
+    LTEXT "Fechou a barra de idiomas.\r\n\r\nPara utilizar a barra de idiomas posteriormente, marque ""Mostrar a barra de idiomas na área de trabalho"" na caixa de diálogo Configurações da barra de idiomas, no painel de controle dos Serviços de Texto.", -1, 5, 5, 190, 60
```

Comment:
Cc @katahiromz : I don't think that we have to use `\r\n` escapes for newlines in resources for dialogs, just `\n` should be enough.

Reply:
You can test `"test\n\ntest"` with Resource Hacker.

Reply:
Note that these \r\n are also in the english translation, they would need to be replaced by \n in a separate PR/commit (unrelated to the present PR though.)